folium/map.py

```diff
@@ -242,16 +242,15 @@ class Marker(MacroElement):
                popup=folium.Popup('Mom & Pop Arrow Shop >>', parse_html=True))
     """
     _template = Template(u"""
-            {% macro script(this, kwargs) %}
-
-            var {{this.get_name()}} = L.marker(
-                [{{this.location[0]}}, {{this.location[1]}}],
-                {icon: new L.Icon.Default(),
-                    {{this.options}})
-                )
-                .addTo({{this._parent.get_name()}});
-            {% endmacro %}
-            """)
+        {% macro script(this, kwargs) %}
```

Comment:
Current convention is to also accept kwargs in the script macro. Not sure what happens if you remove it. But since we use this everywhere I would leave it as is. If there are no kwargs than it will just be empty.